waspc/src/Generator/PackageJsonGenerator.hs

```diff
@@ -63,3 +64,9 @@ toPackageJsonDependenciesString deps =
   "\"dependencies\": {"
     ++ intercalate ",\n  " (map (\dep -> "\"" ++ ND._name dep ++ "\": \"" ++ ND._version dep ++ "\"") deps)
     ++ "\n}"
+
+toPackageJsonDevDependenciesString :: [ND.NpmDependency] -> String
```

Comment:
While both functions are very short so duplication is not terrible, it would still be nice to remove it! Let's remove it. I suggest you create another function that will capture the common part and then you use it in these two functions.

Reply:
Absolutely, take a look again!